interface/modules/custom_modules/oe-module-weno/ModuleManagerListener.php

```diff
@@ -98,6 +99,23 @@ private function install($modId, $currentActionStatus): mixed
         return $currentActionStatus;
     }
 
+    /**
+     * @param $modId
+     * @param $currentActionStatus
+     * @return mixed
+     */
+    private function help_requested($modId, $currentActionStatus): mixed
+    {
+        // must call a script that implements a dialog to show help.
+        // I can't find a way to override the Laminas UI except using a dialog.
+        try {
+            include 'show_help.php';
+        } catch (Exception $e) {
+            return $e->getMessage();
+        }
+        return $currentActionStatus;
+    }
```

Comment:
I could use a little help implementing this because I can't find a way to override the MM UI.
w/o a frame in MM to render help in, I must use a dialog that renders in top if we use an event which'd be nice.
Hoping someone has an idea to get around.
I suppose we could make it a rule for any module to implement in dialog.